app/controllers/api/v1/packages_controller.rb

```diff
@@ -238,7 +238,7 @@ def add_remove_item
       end
 
       def fetch_associated_packages
-        entity = Package.find(params[:id]) if params[:id] # fetch or pallet
+        entity = Package.find_by(params[:id]) if params[:id] # fetch box or pallet
```

Comment:
@abhinav098 it should be `find_by(id: params[:id])`

refer: https://github.com/crossroads/api.goodcity/pull/891#discussion_r376894703

```suggestion
        entity = Package.find_by(id: params[:id]) if params[:id] # fetch box or pallet
```

Reply:
aahh.. my bad, sorry, fixed the typo @swatijadhav .